content/blobstore-agent-password-rotation.md

```diff
@@ -0,0 +1,123 @@
+!!! note
+    Applicable for director version 268.5.0+
+
+# Rotating Blobstore Agent Password
+
+As of director version 268.5.0+, blobstore agent password can be rotated.
+
+### Preconditions
+
+* The Director is in a healthy state.
+* Take note of any ignored VMs. They will be omitted from the VM recreation steps.
```

Comment:
I think we could assume here that operators will find out what ignored vms are or know about them already if they've used ignored vms before. It's like the NATS/Blobstore CA docs in that we put the reminder to look just in case in order to avoid failures due to unintended ignored vms